tests/integration/pilot/istioctl_test.go

```diff
@@ -249,38 +249,38 @@ func TestProxyConfig(t *testing.T) {
 			var args []string
 			g := gomega.NewGomegaWithT(t)
 
-			args = []string{fmt.Sprintf("--namespace=%s", ns.Name()),
-				"pc", "bootstrap", podID}
+			args = []string{"--namespace=dummy",
+				"pc", "bootstrap", fmt.Sprintf("%s.%s", podID, ns.Name())}
```

Comment:
I know this is correct but it doesn't look correct. I don't know why we didn't use `--namespace` in the first place to scope pods instead of the dot notion, but the fact we ignore it entirely is inconsistent with nearly every other istioctl and kubectl command. Do you think its worth adding any extra check to verify `--namespace` isn't set or is at least consistent with the dotted notion?

Reply:
The initial reason for dot notation support was that a debugging flow goes `proxy-status` -> see a proxy that is out of sync -> paste the proxy id into `proxy-config`.